### PR TITLE
COMMERCE-10542

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://issues.liferay.com/browse/COMMERCE-10542

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase [hunk 1]
```diff
@@ -318,4 +318,322 @@ definition {
 		AssertElementPresent(locator1 = "Message#SUCCESS");
 	}
 
+	@description = "This is a test for COMMERCE-10542. Maximum quantity can be added to cart when order is higher then stock quantity"
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase]: ... higher than stock quantity"

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase [hunk 2]
```diff
@@ -318,4 +318,322 @@ definition {
...
+
+		task ("When buyer user adds maximum quantity to the cart") {
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = "8");
+
+			CommerceCheckout.gotoCheckoutViaMiniCart();
+		}
+
+		task ("Then buyer user can do the checkout with the quantity higher then the stock quantity") {
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase]: ... higher than the stock quantity"

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase [hunk 3]
```diff
@@ -318,4 +318,322 @@ definition {
...
+				newAddressName = "Address Name",
+				newAddressZipCode = "Test Zip Code",
+				productName = "U-Joint",
+				productQuantity = "8",
+				productUnitPrice = "24",
+				shippingMethod = "Standard");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-10542. Maximum quantity can be added to cart when order is lower then stock quantity"
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase]: ... is lower than stock quantity"

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase [hunk 4]
```diff
@@ -318,4 +318,322 @@ definition {
...
+
+		task ("When buyer user adds maximum quantity to the cart") {
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = "4");
+
+			CommerceCheckout.gotoCheckoutViaMiniCart();
+		}
+
+		task ("And buyer user can do the checkout with the quantity lower then the stock quantity") {
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase:309]: ... lower than the stock quantity"

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase [hunk 5]
```diff
@@ -318,4 +318,322 @@ definition {
...
+
+		task ("And buyer user adds maximum quantity to the cart") {
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = "4");
+
+			CommerceCheckout.gotoCheckoutViaMiniCart();
+		}
+
+		task ("And buyer user can do the checkout with the quantity lower then the stock quantity") {
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase]: ... lower than the stock quantity"

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase [hunk 6]
```diff
@@ -318,4 +318,322 @@ definition {
...
+				key_productName = "U-Joint",
+				locator1 = "CommerceAccelerators#ADD_TO_CART_BUTTON");
+		}
+
+		task ("Then the order item will not be added to the cart") {
+			Alert.viewErrorMessage(errorMessage = "Unable to add product to the cart.");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-10542. Maximum quantity cannot be added to cart when order is higher then stock quantity"
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase]: ... is higher than stock quantity"

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:forward

**Pull request closed without merging**